cli/src/main/kotlin/commands/UploadResultToPostgresCommand.kt

```diff
@@ -147,5 +149,18 @@ class UploadResultToPostgresCommand : CliktCommand(
         return DriverManager.getConnection(config.url, properties)
     }
 
+    private fun OrtResult.normalizeDependencies() = copy(analyzer = analyzer?.normalizeDependencies())
+
+    private fun AnalyzerRun.normalizeDependencies() =
+        copy(result = result.copy(projects = result.projects.mapTo(sortedSetOf()) { it.normalizeDependencies() }))
+
+    private fun Project.normalizeDependencies() =
+        dependencyGraph?.let {
+            copy(
+                scopeDependencies = it.createScopes(),
+                dependencyGraph = null
+            )
+        } ?: this
```

Comment:
FYI, @oheger-bosch used to have this (by then unused) function in his PR:

https://github.com/oss-review-toolkit/ort/compare/5c5aa4555e6f80b325a5f58fdf444de5005d0664..a96a410847700e18423369864635a538c2623a1b#diff-6bcb52b873b6269f0794bf61bbb9f4d8fa1770d78f1650a613004d1ebd60c3ceL123-L136

Should we restore it for use from here?

Reply:
Sounds good to me, @oheger-bosch do you want to add this function? I would then update this PR afterwards.

Reply:
@mnonnenmacher: Done in #3618.

Reply:
Thanks, I have updated this PR.